src/utils/user.authentication.ts

```diff
@@ -20,21 +20,21 @@ export async function getAccessTokenForUser(
   userRepository: UserRepository,
   credentials: Credentials,
 ): Promise<string> {
-  console.log('getAccessTokenForUser', credentials.password);
+  console.log(credentials.email);
```

Comment:
Can we remove the `console.log` altogether here?

Reply:
good catch removed and squashed into one commit.